htmd/parameterization/util.py

```diff
@@ -180,6 +186,7 @@ def scanDihedrals(mol, ref, dihedrals, outdir, scan_type='qm', mm_minimizer=None
         directories.append(directory)
 
     # Setup and submit QM calculations
+    logger.info('Compute rotamer energies for:')
```

Comment:
Why not before the next loop where the rest of the printing is happening?

Reply:
I moved them together.